components/MoviesList.js

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components/native'
+
+import Movie from './Movie'
+import movieicon from '../assets/movieicon.png'
+
+const URL = "https://api.themoviedb.org/3/movie/popular?api_key=585503bd19f6199055d11458d97ac4cc&language=en-US&page=1"
+
+const MoviesList = () => {
+  const [movies, setMovies] = useState([])
+  
+  useEffect(() => {
+    fetch(URL)
+     .then(res => res.json())
+     .then(json => setMovies(json.results))
+  },[])
+
+  return (
+   <>
+    <Header>
+      <Title>MovieHunter</Title>
+      <MovieImage
+        source={movieicon} />
+    <Container>
+    <PopularText >Popular</PopularText >
+    {movies.map(movie => (
+      <Movie key={movie.id} id={movie.id} poster_path={movie.poster_path} title={movie.title} rating={movie.vote_average} overview={movie.overview}/>
```

Comment:
if you are passing a lot of props then arrange them in the column to make it easier to read 🤓